apps/filebrowser/src/filebrowser/utils.py

```diff
@@ -17,13 +17,14 @@
 import os
 import logging
 from datetime import datetime
+from urllib.parse import urlparse
+
+import redis
```

Comment:
Create a package/class for task-server if not already done. Move all of redis related code in a single class, which exposes high level methods that the rest of the project uses. No other class in the project needs to be aware that redis is being used. Ideally, it should be possible for us to replace redis with something else by modifying 1-2 classes.

Reply:
The Redis-related code is currently located in `filebrowser/utils.py` and `rungunicornserver.py`. As suggested, I plan to create a `TaskServer` class to encapsulate all Redis-related operations. This will make the code cleaner, more modular, and easier to maintain. 

However, due to the urgency of this current task and the need for thorough testing of the refactored code, I will submit a new PR to handle the refactoring.